integration/emr/alluxio-emr.sh

```diff
@@ -237,32 +482,49 @@ If a different Alluxio version is desired, see the -d option.
   -s                A string containing a single character representing what
                     delimiter should be used to split the Alluxio properties
                     provided in the [-p] argument.
+
+  -v                Version of HDFS used as the root UFS. Required when
+                    root UFS is HDFS.
+
 USAGE_END
 )
     echo -e "${USAGE}" >&2
     exit 1
   }
-  local alluxio_tarball=""
+
+  # ordered alphabetically by flag character
   local backup_uri=""
-  local property_delimiter=";"
-  local delimited_properties=""
-  local restore_from_backup_uri=""
+  local client_only="false"
+  local alluxio_tarball=""
+  local execute_synchronous="false"
   local files_list=""
+  local restore_from_backup_uri=""
   local nvme_capacity_usage=""
-  local client_only="false"
+  local delimited_properties=""
+  local property_delimiter=";"
+  local hdfs_version=""
 
   if [[ "$#" -lt "1" ]]; then
     echo -e "No root UFS URI provided"
     print_help 1
   fi
 
   local root_ufs_uri="${1}"
+  # note that since the script args are shifted
+  # the shifted argument needs to be manually added when launching the background process
   shift
-  while getopts "b:cd:f:i:n:p:s:" option; do
-    if [[ -n "${OPTARG-}" ]]; then
-      OPTARG=$(echo -e "${OPTARG}" | tr -d '[:space:]')
-    fi
+
+  while getopts "ehb:cd:f:i:n:p:s:v:" option; do
```

Comment:
-c                Install the alluxio client jars only.？
Where assign value to client_only?

Reply:
https://github.com/Alluxio/alluxio/pull/11545

Reply:
oh shit i didn't add it in here

Reply:
oh wait i dont need to cuz it was already committed...

Reply:
i saw the line difference